test/support/utils.rb

```diff
@@ -18,11 +18,15 @@ def assert_line_num(expected)
 
     def assert_line_text(expected)
       @queue.push(Proc.new {
-        assert_block do
-          @internal_info['backlog'].each do |line|
-            line.include? expected
+        msg = "Expected to include #{expected}}"
```

Comment:
it still doesn't provide much information:

```
  Expected to include => 20}.
  <true> expected but was
  <false>
```

how about this?

```ruby
assert_match(expected, @internal_info['backlog'].join)
```

```
  </=>\ 20/> was expected to be =~
  <"=>#0\tFoo#first_call at /var/folders/yg/hnbymwxd5pn7v94_clc59y6r0000gn/T/debugger20210608-65153-b857w.rb:4 #=> 30\n" +
  "  #1\t<main> at /var/folders/yg/hnbymwxd5pn7v94_clc59y6r0000gn/T/debugger20210608-65153-b857w.rb:23\n">.
```

Reply:
That's a good idea! Thanks 👍

Reply:
https://github.com/ruby/debug/pull/71/files#diff-ee14bf65f14927f7778a246f90c5cec0c581be3e2658c2a8a3726c59db5be3f9R21